src/api/server.ts

```diff
@@ -5,7 +5,6 @@ import * as logger from './logger';
 import * as fs from './fs';
 import * as utils from './utils';
 import { initializeStorage } from './storage';
-import { ISocketServerOptions, SocketServer } from './socket';
 import { loadAverage, networkUtilization } from './system';
```

Comment:
I don't think that loadAverage and networkUtilization functions are still required. They don't need to be imported here and the whole file system.ts can be deleted.